pkgs/tools/misc/clipbuzz/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, fetchFromSourcehut, zig, libX11, libXfixes }:
+
+stdenv.mkDerivation rec {
+  pname = "clipbuzz";
+  version = "2.0.0";
+
+  src = fetchFromSourcehut {
+    owner = "~cnx";
+    repo = pname;
+    rev = version;
+    sha256 = "1qn98bwp7v7blw4v0g4pckgxrky5ggvq9m0kck2kqw8jg9jc15jp";
+  };
+
+  nativeBuildInputs = [ zig ];
+  buildInputs = [ libX11 libXfixes ];
+
+  preBuild = "export HOME=$TMPDIR";
+  installPhase = ''
+    zig build -Drelease-safe -Dcpu=baseline --prefix $out install
+  '';
+
+  meta = with lib; {
+    description = "Buzz on new X11 clipboard events";
+    inherit (src.meta) homepage;
```

Comment:
Please set the homepage explicitly. I am not going to copy the url together from multiple variables.

Reply:
Sorry, I saw this in other plan and thought it was conventional :smile:

Reply:
When I get around I am probably removing that from all files in nixpkgs. It is a nice little shortcut but while reviewing it is pretty tedious to visit the website.